raiden-ts/src/config.ts

```diff
@@ -115,6 +115,8 @@ export const PartialRaidenConfig = t.readonly(
 );
 export interface PartialRaidenConfig extends t.TypeOf<typeof PartialRaidenConfig> {}
 
+const rollupChainIds = new Set([421611, 42161]);
```

Comment:
Maybe add names so we know what they should refer to? 🤷🏾

Reply:
:heavy_check_mark: